python/prod/worker.py

```diff
@@ -0,0 +1,66 @@
+import os
+import sys
+import platform
+import subprocess
+from pathlib import Path
+
+# User packages
+PKGS_PATH = "site/wwwroot/.python_packages"
+VENV_PKGS_PATH = "site/wwwroot/worker_venv"
+
+PKGS_36 = "lib/python3.6/site-packages"
+PKGS = "lib/site-packages"
```

Comment:
assuming `/` instead of `os.path.sep` or `os.path.join` is ok as this will only be used in Azure.

Reply:
Yes, that's right. I wanted to define these as constants to make it clear that this is only for the Linux environment in Azure.